examples/text_classification/tc_multi_languages_transformers.ipynb

```diff
@@ -0,0 +1,329 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "*Copyright (c) Microsoft Corporation. All rights reserved.*\n",
+    "\n",
+    "*Licensed under the MIT License.*\n",
+    "\n",
+    "# Text Classification of Multi Language Datasets using Transformer Model"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import scrapbook as sb\n",
+    "import pandas as pd\n",
+    "\n",
+    "from tempfile import TemporaryDirectory\n",
+    "from utils_nlp.common.timer import Timer\n",
+    "from sklearn.metrics import classification_report\n",
+    "from utils_nlp.models.transformers.sequence_classification import *\n",
```

Comment:
in other notebooks we try not to add `import *`, instead add the method that we are going to use

Reply:
Updated and tested.